challenge/dns01/fqdn.go

```diff
@@ -6,15 +6,6 @@ import (
 	"github.com/miekg/dns"
 )
 
-// ToFqdn converts the name into a fqdn appending a trailing dot.
-func ToFqdn(name string) string {
```

Comment:
Removing an export is technically a breaking change.

Better leave it in place and deprecate it:

```go
// ToFqdn converts the name into a fqdn appending a trailing dot.
//
// Deprecated: Use [github.com/miekg/dns.Fqdn] directly.
var ToFqdn = dns.Fqdn
```